### PR TITLE
Initial Structure Setup

### PR DESCRIPTION
**Pull request by RajVarsani**
## Changes: 

- Set up folder structure hierarchies
- Add general dependencies
- Code practices guidelines

### TO DO :
- add style variables and guidelines once the design is ready.

Suggestions and inputs are welcome.

### CODE REVIEW

**Review by escapedcat — COMMENTED** [review]
Tried and it starts up and I don't get any errors. Good first step!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# getalby.com (Front-End)
+
+## Getting Started
+
+Make sure to run `npm install` and `npm run prepare` after cloning the repository.
```
[the remark this answers is not in the document]
Comment (RajVarsani) [reply README.md:5]: Yep, that makes sense. Thanks for pointing it out. Will update commands to yarn and push it.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Getting Started
+
+> Make sure to run `yarn install` and `yarn prepare` after cloning the repository.
+
+Run the development server:
+
+```bash
+yarn dev
+# or
```
Comment (escapedcat) [inline README.md:11]: This can be removed :P

File: package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "pretty": "prettier --write .",
+    "prepare": "husky install"
+  },
+  "dependencies": {
+    "next": "12.3.1",
```
Comment (escapedcat) [inline package.json:14]: Do we prefer `^` for versions here?

**Review by escapedcat — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (RajVarsani) [reply package.json:14]: On `create next app`, it came as default without `^`. Ig, we can go with the default, right? Let me know what you think.

Comment (escapedcat) [reply package.json:14]: We have a `yarn.lock` to fix deps. Let's add the `^`. But it's non-blocking as well.

**Pull request merged**